include/openssl/conf.h

```diff
@@ -90,6 +67,20 @@ int CONF_dump_bio(LHASH_OF(CONF_VALUE) *conf, BIO *out);
 
 DEPRECATEDIN_1_1_0(void OPENSSL_config(const char *config_name))
 
+DEPRECATEDIN_3(CONF_VALUE *_CONF_new_section(CONF *conf, const char *section))
+DEPRECATEDIN_3(CONF_VALUE *_CONF_get_section(
+                    const CONF *conf, const char *section))
+DEPRECATEDIN_3(STACK_OF(CONF_VALUE) *_CONF_get_section_values(
+                    const CONF *conf, const char *section))
+DEPRECATEDIN_3(int _CONF_add_string(
+                    CONF *conf, CONF_VALUE *section, CONF_VALUE *value))
+DEPRECATEDIN_3(char *_CONF_get_string(const CONF *conf, const char *section,
+                    const char *name))
+DEPRECATEDIN_3(long _CONF_get_number(const CONF *conf, const char *section,
+                    const char *name))
+DEPRECATEDIN_3(int _CONF_new_data(CONF *conf))
+DEPRECATEDIN_3(void _CONF_free_data(CONF *conf))
+
```

Comment:
Introducing them here is the wrong move.

Reply:
I would actually just remove `conf_api.h`, or rather move it to a private place.

Reply:
I would really like to just remove them, but when I tried to do that before some members of the team pushed back against it (including yourself; see #3402). @t-j-h has also said that functions declared in header files are public.  If views have changed, great.

Reply:
If you read what I wrote in #3402, you will notice that I pre-approved the idea for the next major release. I pushed back because it was proposed at a time we were busy with the next minor release.

So no, my view hasn't changed.

Reply:
i removed the functions.